tests/Form/DataTransformer/ModelToIdPropertyTransformerTest.php

```diff
@@ -115,34 +119,36 @@ public function getReverseTransformMultipleTests(): iterable
         $entity3->setBar('example3');
 
         yield [[], null, $entity1, $entity2, $entity3];
-        yield [[], false, $entity1, $entity2, $entity3];
         yield [[$entity1], [123, '_labels' => ['example']], $entity1, $entity2, $entity3];
         yield [[$entity1, $entity2, $entity3], [123, 456, 789, '_labels' => ['example', 'example2', 'example3']], $entity1, $entity2, $entity3];
     }
 
     /**
+     * @param mixed $params
+     *
      * @dataProvider getReverseTransformMultipleInvalidTypeTests
      */
-    public function testReverseTransformMultipleInvalidTypeTests(array $expected, $params, string $type): void
+    public function testReverseTransformMultipleInvalidTypeTests($params, string $type): void
     {
         $this->expectException(\UnexpectedValueException::class);
         $this->expectExceptionMessage(sprintf('Value should be array, %s given.', $type));
 
         $transformer = new ModelToIdPropertyTransformer($this->modelManager, Foo::class, 'bar', true);
 
-        $result = $transformer->reverseTransform($params);
-        $this->assertInstanceOf(ArrayCollection::class, $result);
-        $this->assertSame($expected, $result->getValues());
+        $transformer->reverseTransform($params);
     }
 
+    /**
+     * @phpstan-return array<array{mixed, string}>
+     */
     public function getReverseTransformMultipleInvalidTypeTests(): array
     {
         return [
-            [[], true, 'boolean'],
-            [[], 12, 'integer'],
-            [[], 12.9, 'double'],
-            [[], '_labels', 'string'],
-            [[], new \stdClass(), 'object'],
+            [true, 'boolean'],
+            [12, 'integer'],
+            [12.9, 'double'],
```

Comment:
```suggestion
            [12.9, 'float'],
```

Reply:
Test are failing with this change, because `gettype` is returning double.
According to the doc:
```
(for historical reasons "double" is returned in case of a float, and not simply "float")
``